### PR TITLE
Issue #1217 Update to latest galvan

### PR DESCRIPTION
**Pull request by gibsong**
…tems.  This is a temporary fix until the Galvan #61 bug is fixed

### CODE REVIEW

File: clustered/integration-test/src/test/java/org/ehcache/clustered/TerminatedServerTest.java [hunk 1]
```diff
@@ -102,6 +103,18 @@
...
 
+  /**
+   * The tests in this class are failing on Windows.  There is a windows bug in the Galvan code.
+   * This is a temporary fix until the Galvan bug is resolved.  When the code below is run
+   * on a Windows OS all the tests will be skipped.
+   * The Galvan issue is Terracotta-OSS/galvan#61
+  **/
+  @BeforeClass
+  public static void checkOS()
+  {
```
Comment (lorban) [inline clustered/integration-test/src/test/java/org/ehcache/clustered/TerminatedServerTest.java]: Code style please! Opening curly brace should be on the method line.

Comment (gibsong) [inline clustered/integration-test/src/test/java/org/ehcache/clustered/TerminatedServerTest.java]: shoot!  sorry about the formatting.  I am removing this method, as I fixed the bug in Galvan.

**Comment by gibsong** [comment]
Louis, yes this code will no longer need to be merged once the new version of Galvan is released.  I can close it once that happens.

**Comment by gibsong** [comment]
The Galvan bug has been fixed, and the fix was released with the new version of Galvan 1.0.2-beta2.  I had to make some unit test and gradle build changes to integrate the Galvan changes.

**Pull request merged**